pyth_observer/events.py

```diff
@@ -555,3 +557,54 @@ def get_event_details(self) -> Tuple[str, List[str]]:
             f"CoinGecko Price Chart: {url}",
         ]
         return title, details
+
+
+class PriceDeviationCrosschain(PriceAccountValidationEvent):
+    """
+    This alert is supposed to fire when a cross-chain price feed deviates
+    from Solana price feed by a specified threshold.
+    """
+
+    error_code: str = "price-deviation-cross-chain"
+    threshold = 20
+
+    def is_valid(self) -> bool:
+        # check if cross-chain price exists
+        if not self.crosschain_price:
+            return True
+
+        trading = (
+            self.price_account.aggregate_price_info.price_status
+            == PythPriceStatus.TRADING
+        )
+        pyth_price = self.price_account.aggregate_price_info.price
+
+        if not trading or pyth_price == 0:
+            return True
+
+        delta = abs(pyth_price - self.crosschain_price["price"])
+        self.crosschain_deviation = delta / self.crosschain_price["conf"]
```

Comment:
i think we should do a % threshold without the confidence interval. I don't think the CI is super meaningful when we're comparing two aggregate prices from different times (which is what this alert will do).

Reply:
changed to 5% default threshold and it's configurable through env var `PYTH_OBSERVER_PRICE_DEVIATION_CROSSCHAIN`